server/devdocs/index.js

```diff
@@ -1,20 +1,22 @@
 /**
  * Extenal dependencies
  */
-var express = require( 'express' ),
-	fs = require( 'fs' ),
-	fspath = require( 'path' ),
-	marked = require( 'marked' ),
-	lunr = require( 'lunr' ),
-	find = require( 'lodash/find' ),
-	escapeHTML = require( 'lodash/escape' );
+import express from 'express';
+import fs from 'fs';
+import fspath from 'path';
+import marked from 'marked';
+import lunr from 'lunr';
+import find from 'lodash/find';
+import escapeHTML from 'lodash/escape';
+import striptags from 'striptags';
 
 /**
  * Internal dependencies
  */
-var	config = require( 'config' ),
-	root = fs.realpathSync( fspath.join( __dirname, '..', '..' ) ),
-	searchIndex = require( 'devdocs/search-index' ),
+import config from 'config';
+import searchIndex from 'devdocs/search-index';
+
+var	root = fs.realpathSync( fspath.join( __dirname, '..', '..' ) ),
```

Comment:
I think this might be the case of the single space matching up to the tab
column...let me have another look today

On Friday, March 4, 2016, Andrew Duthie <notifications@github.com> wrote:

> In server/devdocs/index.js
> <https://github.com/Automattic/wp-calypso/pull/3555#discussion_r55034836>:
>
> >
> >  /**
> >   * Internal dependencies
> >   */
> > -var	config = require( 'config' ),
> > -	root = fs.realpathSync( fspath.join( __dirname, '..', '..' ) ),
> > -	searchIndex = require( 'devdocs/search-index' ),
> > +import config from 'config';
> > +import searchIndex from 'devdocs/search-index';
> > +
> > +var	root = fs.realpathSync( fspath.join( __dirname, '..', '..' ) ),
>
> Checked this again, it is a single space, not sure why the diff shows it
> like this...
>
> It displays as though it were a single space in my editor, but it's
> definitely a tab.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Automattic/wp-calypso/pull/3555/files#r55034836>.
>